apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/parser/gql/Schema.kt

```diff
@@ -0,0 +1,56 @@
+package com.apollographql.apollo.compiler.parser.gql
+
+/**
+ * a very thin wrapper around a schema GQLDocument
+ *
+ * It serves as a common ground between GQLDocument and IntrospectionSchema
+ *
+ * Schema should always contain all types, including the builtin ones
+ */
+class Schema(
```

Comment:
nit: should we call it `GQLSchema`?

Reply:
I'd like to keep the `GQL` prefix for everything that is in the `GQLNode` type hierarchy and `Schema` doesn't inherit from `GQLNode`.

There should be only 2 instances of `Schema`: one for introspection and this one so I think having `Schema` and `IntrospectionSchema` isn't too confusing but I'm happy to think of another prefix if needed.